content/applications/inventory_and_mrp/inventory/product_management/product_replenishment/uom.rst

```diff
@@ -40,17 +44,12 @@ the :guilabel:`Uom` column of the :guilabel:`Units of Measure Categories` page.
 reference unit as a base for any new units.
 
 To create a new unit, first select the correct category. For example, to sell a product in a box of
-six units, click on the :guilabel:`Unit` category line. Next, click :guilabel:`Edit`. After that,
-click :guilabel:`Add a line`. Then, in the :guilabel:`Unit of Measure` field, title the new unit
-`Box of 6`. In the :guilabel:`Type` field, select :guilabel:`Bigger than the reference Unit of
-Measure`. In the :guilabel:`Ratio` field, enter `6.00000` since a box of six is six times bigger
-than the reference unit (`1.00000`). Finally, click :guilabel:`Save`.
-
-Click on the :guilabel:`Unit` category.
+six units, click on the :guilabel:`Unit` category line.
 
-Click :guilabel:`Add a line`. As an example, we will create a Box of 6 units that we will use for
-the Egg product. The box of 6 is 6 times bigger than the reference unit of measure for the category
-which is “Units” here.
+After that, click :guilabel:`Add a line`. Then, in the :guilabel:`Unit of Measure` field, title the
+new unit `Box of 6`. In the :guilabel:`Type` field, select :guilabel:`Bigger than the reference Unit
+of Measure`. In the :guilabel:`Ratio` field, enter `6.00000`, since a box of six is six times bigger
+than the reference unit (`1.00000`). Finally, click :guilabel:`Save`.
```

Comment:
```suggestion
On the category page that appears, click :guilabel:`Add a line` in the :guilabel:`Units of Measure`
tab.

Then, in the :guilabel:`Unit of Measure` field, title the new unit: `Half Dozen`.

In the :guilabel:`Type` field, select :guilabel:`Bigger than the reference Unit of Measure`.

In the :guilabel:`Ratio` field, enter `6.00000`, since a box of six is six times *bigger* than the
reference unit (`1.00000`).
```
broke up the chunky paragraph into more digestible steps.

also, opted for "half dozen" or "Box of 6" to fall in line with the previously established "Dozen" unit in the category.

And, is there a reason why the UNSPSC column/field was glossed over? It might be beneficial to briefly touch on what that means/how it's useful/if it's necessary/etc,.

Reply:
UPDATE: I realized that if "Box of 6" is changed to "Half Dozen" it would require a series of screenshots to be recaptured. If this proves too cumbersome, you can ignore the suggestion.